samples/webapps/jaxrs_basic/pom.xml

```diff
@@ -39,13 +39,15 @@
             </plugin>
             <plugin>
                 <artifactId>maven-war-plugin</artifactId>
+                <version>${maven.war.plugin.version}</version>
                 <configuration>
                     <packagingExcludes>WEB-INF/lib/*.jar</packagingExcludes>
                     <warName>${project.artifactId}</warName>
                 </configuration>
             </plugin>
             <plugin>
                 <artifactId>maven-eclipse-plugin</artifactId>
+                <version>2.10</version>
```

Comment:
Shall we move this to parent pom and refer the version here

Reply:
Fixed with 
a9a4c30